vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>"The <code>product-image</code> mode always returns 4 1024x1024 images."</p><p>should be </p><p>"The <code>product-image</code> mode always returns four 1024x1024 images." to reduce reader confusion with sequential Arabic numerals.</p><p> </p><p>"<strong>perfect</strong> for advertisements and online shopping" should be "<strong>great</strong> for advertisements and online shopping".</p><p> </p><p>"<strong>We will generate</strong> a foreground product mask for you." should be "<strong>Imagen</strong> <strong>generates</strong> the foreground product mask automatically."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Updated.